README.adoc

```diff
@@ -258,9 +254,7 @@ Rerun the Maven build. You see a test failure occur.
 
 == Great work! You're done!
 
-You just built and tested a RESTful client interface to invoke a remote service with MicroProfile Rest Client and Open Liberty.
-
-For more information on the MicroProfile Rest Client, see the http://download.eclipse.org/microprofile/microprofile-rest-client-1.0/microprofile-rest-client.pdf[MicroProfile Rest Client 1.0 spec].
+You just built and tested invoking a remote service using a template interface with MicroProfile Rest Client and Open Liberty.
```

Comment:
What about, "You just invoked a remote service by using a template interface with MicroProfile Rest Client and Open Liberty."?

I think that "invoking" used as a noun could potentially be confusing.